requires-install.txt

```diff
@@ -5,3 +5,5 @@ dash_core_components==2.0.0
 dash_table==5.0.0
 importlib-metadata==4.8.3;python_version<"3.7"
 contextvars==2.4;python_version<"3.7"
+typing_extensions>=4.1.1
+stringcase>=1.2.0
```

Comment:
do we need `stringcase` in `requires-install`? Couldn't it go in `requires-dev`?

Reply:
It's needed for the old runtime component loader :disappointed:

Reply:
haha ok - we should put a deprecation warning on runtime component generation, and remove it whenever we decide to do a Dash 3.0